hvpy/facade.py

```diff
@@ -298,3 +299,86 @@ def downloadScreenshot(id: int) -> Union[bytes, str, Dict[str, Any]]:
     """
     params = downloadScreenshotInputParameters(id=id)
     return execute_api_call(input_parameters=params)
+
+
+@add_shared_docstring(queueMovieInputParameters)
+def queueMovie(
+    startTime: datetime,
+    endTime: datetime,
+    layers: str,
+    events: str,
+    eventsLabels: bool,
+    imageScale: float,
+    format: Optional[str] = "mp4",
+    frameRate: Optional[str] = "15",
+    maxFrames: Optional[str] = None,
+    scale: Optional[bool] = None,
+    scaleType: Optional[str] = None,
+    scaleX: Optional[float] = None,
+    scaleY: Optional[float] = None,
+    movieLength: Optional[float] = None,
+    watermark: Optional[bool] = True,
+    width: Optional[str] = None,
+    height: Optional[str] = None,
+    x0: Optional[str] = None,
+    y0: Optional[str] = None,
+    x1: Optional[str] = None,
+    y1: Optional[str] = None,
+    x2: Optional[str] = None,
+    y2: Optional[str] = None,
+    callback: Optional[str] = None,
+    size: Optional[int] = None,
+    movieIcons: Optional[int] = None,
+    followViewport: Optional[int] = None,
+    reqObservationDate: Optional[datetime] = None,
+) -> Union[bytes, str, Dict[str, Any]]:
+    """
+    Queue a movie for download from the helioviewer.org API.
+
+    Parameters
+    ----------
+    {Insert}
+    Examples
+    --------
+    >>> from hvpy import queueMovie
+    >>> queueMovie(
+    ...     startTime=datetime(2022, 7, 21, 12, 12, 12),
+    ...     endTime=datetime(2022, 7, 22, 12, 12, 12),
+    ...     layers="[12,7,22],[13,7,11]",
+    ...     events="[AR,HMI_HARP;SPoCA,1],[CH,all,1]",
+    ...     eventsLabels=False,
+    ...     imageScale=2.44,
+    ... )
+    {'id': ...}
```

Comment:
What does the full response look like out of curiosity?

Reply:
```
{
   'id': 'gxRN5', 
   'eta': 274, 
   'queue': 0, 
   'token': '5d537e2d54dbdc725cbb26dda830e2f0'
}
```